packages/create-nuxt3-app/README.md

```diff
@@ -1,4 +1,61 @@
 # create-nuxt3-app
 
-See more information about usage on the main [ReadMe](../../README.md).
+[![npm version][npm-version-src]][npm-version-href]
+[![npm downloads][npm-downloads-src]][npm-downloads-href]
```

Comment:
this count is outdated!

please ctrl + shift + f and ensure that `sidebase` is replaced with create-nuxt3-app (:

Reply:
It should not be. I changed the values at the bottom of the ReadMe to reflect the correct repo!